src/saving/ThriveArchiveObjectType.cs

```diff
@@ -218,6 +218,7 @@ public enum ThriveArchiveObjectType : uint
     ComponentStaticBodyMarker = 4302,
     ComponentTimedLife = 4303,
     ComponentWorldPosition = 4304,
+    ComponentCellDivisionCollisionDisabler = 4305,
```

Comment:
You cannot add a duplicate value here, this will blow up saves. You need to add the new value at the end (and sadly not grouped with the other components).